src/lib/tsreader/MemoryBuffer.cpp

```diff
@@ -56,14 +56,17 @@ bool CMemoryBuffer::IsRunning()
 
 void CMemoryBuffer::Clear()
 {
-  P8PLATFORM::CLockObject BufferLock(m_BufferLock);
+  std::lock_guard<std::mutex> BufferLock(m_BufferLock);
   std::vector<BufferItem *>::iterator it = m_Array.begin();
```

Comment:
This line can be removed, too.

Reply:
We can only make minimal changes to anything in the lib folder.